src/EnrollCallAndSmsController.js

```diff
@@ -201,6 +202,7 @@ function (Okta, FormController, Footer, PhoneTextBox, CountryUtil, FormType, Key
           placeholder: Okta.loc('mfa.phoneNumber.ext.placeholder', 'login'),
           className: 'enroll-call-extension',
           name: 'phoneExtension',
+          input: TextBox,
           type: 'text'
```

Comment:
We do actually. Courage chokes if it's not present on some inputs (particularly password types)